### PR TITLE
Use pseudo patrons for Folio Page/MediatedPage requests

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: config/settings.yml
Comment (marlo-longley) [inline config/settings.yml:377]: Since these are all the same key/value (`folio_hold_pseudopatron: 5eb559e2-e986-4158-bffd-16bb6e312ca1`) is there a way to set it once, and reference a variable from then on? This could easier to read and easier to quickly understand that these all use the same pseudopatron, rather than trying to read these UUIDs multiple times.

Comment (jcoyne) [reply config/settings.yml:377]: It's not going to be all the same, Sarah just hadn't created them all just yet.

Comment (jcoyne) [reply config/settings.yml]: These UUIDs have now been updated.

Comment (thatbudakguy) [reply config/settings.yml]: Could we just store pseudopatron barcodes instead? That's what https://github.com/sul-dlss/folio-graphql/pull/88 assumes we will do when we migrate the config to FOLIO, and it's what we've done with other values that could otherwise be UUIDs (like location codes) so it's easier to edit them and see what they mean.

Comment (jcoyne) [reply config/settings.yml]: @thatbudakguy it's possible yes, but this would introduce another API call to resolve the user_id.  API calls are not reliable, so handling errors introduces unnecessary complexity.  Are we sure we want to add additional failure modes so that we can avoid looking at UUIDs?

Comment (jcoyne) [reply config/settings.yml]: In post-standup today we determined that this optimization could wait for later

**Comment by marlo-longley** [comment]
We decided to merge this and revisit once work is merged that allows us to pull data about pseudopatrons directly from FOLIO (once we can use our graphql endpoint in sul-requests). At that point we will be referencing patron barcodes rather than the UUIDs we see here.

**Review by marlo-longley — APPROVED** [review]

**Pull request merged**
